src/components/ScrollableContainer/ScrollableContainer.test.js

```diff
@@ -0,0 +1,112 @@
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+            <p>
+              Ullamco reprehenderit in irure officia dolore anim eiusmod labore
+              duis ea laborum ex. Reprehenderit consequat officia ea id ex
+              exercitation et sit et. Velit velit aliqua occaecat quis occaecat.
+              Enim incididunt est velit pariatur adipisicing labore dolore anim
+              cillum.
+            </p>
+          </BodyUI>
+        }
+        footer={<FooterUI className="TESTING__FOOTER">Footer</FooterUI>}
+      />
+    )
+
+    const containerScroll = container.querySelector('.c-ScrollableContainer')
+    const header = container.querySelector('.ScrollableContainer__Header')
+    const body = container.querySelector('.ScrollableContainer__Body')
+    const footer = container.querySelector('.ScrollableContainer__Footer')
+
+    // Elements present
+    expect(containerScroll).toBeInTheDocument()
+    expect(header).toBeInTheDocument()
+    expect(body).toBeInTheDocument()
+    expect(footer).toBeInTheDocument()
+
+    // Passing custom classnames
+    expect(header.classList.contains('TESTING__HEADER')).toBeTruthy()
```

Comment:
this can be replaced with `expect(header).toHaveClass('TESTING__HEADER')` (and the same for assertions below). It would be giving more meaningful error messages in case it's not true, instead of `expected false to be truthy` or something like this :smile:

Reply:
I always forget there are so many matchers! thanks for the reminder